src/page-objects/base.ts

```diff
@@ -17,6 +17,11 @@ import { getElementCenter } from './utils';
 import { ElementRect } from './types';
 import { waitForTimerAndAnimationFrame } from './browser-scripts';
 
+interface ExtendedWindow extends Window {
+  __liveAnnouncements?: string[];
+}
+declare const window: ExtendedWindow;
```

Comment:
Thanks, will update 👍

Reply:
It redefines only module-level value.

If following my comment above: https://github.com/cloudscape-design/browser-test-tools/pull/35#discussion_r1101280742, it will be properly scoped